core/server/common/src/main/java/alluxio/master/audit/AuditContext.java

```diff
@@ -0,0 +1,23 @@
+package alluxio.master.audit;
+
+import java.io.Closeable;
+
+/**
+ * Context for Alluxio audit logging.
+ */
+public interface AuditContext extends Closeable {
+
+  /**
+   * Set to true if the operation associated with this {@link AuditContext} is allowed, false otherwise
+   *
+   * @param allowed true if operation is allowed, false otherwise
+   * @return {@link AuditContext} instance itself
+   */
+  AuditContext setAllowed(boolean allowed);
+
+  /**
+   * Operation to perform when this {@link AuditContext} instance is closed
```

Comment:
I don't think this comment adds any new information - we can remove it to inherit the superclass javadoc